src/main/ruby/core/truffle/process_operations.rb

```diff
@@ -0,0 +1,520 @@
+# Copyright (c) 2015, 2018 Oracle and/or its affiliates. All rights reserved. This
```

Comment:
Looks like process_mirror.rb was not removed.

Reply:
Good catch.